docs/user/config-file/v2.rst

```diff
@@ -803,7 +803,9 @@ Schema
 
 You can see the complete schema
 `here <https://github.com/readthedocs/readthedocs.org/blob/main/readthedocs/rtd_tests/fixtures/spec/v2/schema.json>`_.
+This schema is available at `Schema Store`_, use it with your favorite editor for validation and autocompletion.
```

Comment:
It's currently pointing to `master` branch; so I'm not sure if we need to update this on that website.

Reply:
GitHub does the redirect automatically